BUILDTARGETS.md

```diff
@@ -2,8 +2,35 @@
 
 The Unity Buy SDK currently supports the following build targets:
 
+* [Android](#Android)
 * [iOS](#ios)
 
+# Android
+
+### Requirements
+
+The Unity Buy SDK requires Android games to have a minimum API level of Android 4.4 'Kit Kat' (API Level 19)
+
+### Building
+
+The library comes in a form of a `.AAR` file. In doing so we have provided a [mainTemplate.gradle](Assets/Plugins/Android/mainTemplate.gradle) that will be used by Unity when creating a Gradle project.
+
+The Gradle file includes the dependencies used by the libarary. If you would like to use your own Gradle file, add the following dependencies:
```

Comment:
The only way to mitigate this would be to include dependencies into the .AAR which might not play nice with other Android plugins who use the same dependencies. I think for now it is easier to support 5.6 and up